packages/components/src/components/files/FileTree.tsx

```diff
@@ -104,12 +104,13 @@ const nodeIsEmpty = (id: string): boolean => {
 };
 
 const Header = props => {
-    const { style, onSelect, node, customStyles, checked, handleCheckbox, useFileIconCls } = props;
+    const { style, onSelect, node, customStyles, checked, handleCheckbox, useFileIconCls, emptyDirectoryText, allowMultiSelect } = props;
     const isDirectory = node.children !== undefined;
     const icon = isDirectory ? (node.toggled ? faFolderOpen : faFolder) : faFileAlt;
+    const activeColor = (node.active && !allowMultiSelect) ? '#2980b9' : '#777'; // $brand-primary and $gray-light
```

Comment:
Was there somewhere that this #777 was being used as the default color before or is that just being added with this story? If it wasn't a previous default, I wonder if we should allow it to be set as `undefined` in that case since an inline style like this would trump any CSS that an app might have tried to set for this node.

Reply:
Also, the text color of `#2980b9` is the "sky" theme color, but what about if the LK server has a different theme set? I wonder if instead, we could use a css class like `lk-text-theme-dark` which is exposed from the `_labkey.scss` file as a way to make the theme text color available for use via a css class name.

Reply:
Good point, done!